src/drivers/uavcan/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ if(CONFIG_ARCH_CHIP)
 		set(UAVCAN_TIMER 1)
 	elseif(${CONFIG_ARCH_CHIP} MATCHES "stm32h7")
 		set(UAVCAN_DRIVER "stm32h7")
+		set(UAVCAN_TIMER 5) # The default timer is TIM5
 		if (DEFINED config_uavcan_timer_override)
 		set (UAVCAN_TIMER  ${config_uavcan_timer_override})
```

Comment:
Please indent this line. Also do you know How to squash the commits? Please squash the commits to the same files to one commit **PER FILE**. changed.

Reply:
`	elseif(${CONFIG_ARCH_CHIP} MATCHES "stm32h7")
		set(UAVCAN_DRIVER "stm32h7")
			set(UAVCAN_TIMER 5) # The default timer is TIM5
		if (DEFINED config_uavcan_timer_override)
			set (UAVCAN_TIMER  ${config_uavcan_timer_override})
		endif()`
indent like this ? what ablout the "set (UAVCAN_TIMER  ${config_uavcan_timer_override})"? Should I indent it too?

Reply:
It is line 50
https://github.com/PX4/Firmware/blob/f222d046fccc2a75b49d8c37779ee459eac2cf86/src/drivers/uavcan/CMakeLists.txt#L50